File: RepoTransBench/source_projects/Java/yandex-qatools_postgresql-embedded/src/test/java/de/flapdoodle/embed/process/store/IMutableArtifactStoreTest.java
package de.flapdoodle.embed.process.store;

import de.flapdoodle.embed.process.config.store.IDownloadConfig;
import org.junit.Test;

import static org.junit.Assert.*;

public class IMutableArtifactStoreTest {
    private static class TestMutableArtifactStore implements IMutableArtifactStore {
        public boolean configSet = false;
        @Override public void setDownloadConfig(IDownloadConfig downloadConfig) { configSet = true; }
        @Override public void removeFileSet(Object d, Object fs) {}
        @Override public boolean checkDistribution(Object d) {return false;}
        @Override public Object extractFileSet(Object d) {return null;}
    }

    @Test
    public void testSetDownloadConfig() {
        TestMutableArtifactStore store = new TestMutableArtifactStore();
        assertFalse(store.configSet);
        store.setDownloadConfig(null);
        assertTrue(store.configSet);
    }
}

File: RepoTransBench/target_projects/Java/Go/yandex-qatools_postgresql-embedded/tests/original/imutable_artifact_store_test.go
package original

import "testing"

type testIDownloadConfig2 struct {}

type TestMutableArtifactStore struct {
	configSet bool
}
func (t *TestMutableArtifactStore) SetDownloadConfig(cfg *testIDownloadConfig2) { t.configSet = true }
func (t *TestMutableArtifactStore) RemoveFileSet(d, fs interface{})              {}
func (t *TestMutableArtifactStore) CheckDistribution(d interface{}) bool         { return false }
func (t *TestMutableArtifactStore) ExtractFileSet(d interface{}) interface{}     { return nil }

func TestSetDownloadConfig(t *testing.T) {
	store := &TestMutableArtifactStore{}
	if store.configSet {
		t.Errorf("configSet should be false initially")
	}
	store.SetDownloadConfig(nil)
	if !store.configSet {
		t.Errorf("configSet should be true after SetDownloadConfig()")
	}
}